storybook/CustomPropTable.js

```diff
@@ -0,0 +1,123 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import PropVal from '@storybook/addon-info/dist/components/PropVal'
+import PrettyPropType from '@storybook/addon-info/dist/components/types/PrettyPropType'
+
+const Table = props => <table {...props} />
+const Td = props => <td style={{paddingRight: 10, paddingTop: 10, textAlign: 'left', verticalAlign: 'top'}}
```

Comment:
It would be possible by wrapping component CustomPropTable and reseting props.style for all table components. The current implementation is closer to the original which only uses inline styling. IMO currently we are not urged to move to styled components.